README.md

```diff
@@ -49,12 +49,14 @@ to the `accessories` section. As an example:
         "accessory": "PurpleAirSensor",
         "sensor": "12345",
         "name": "PurpleAir Indoor Sensor",
+        "conversoin": "AQandU",
```

Comment:
conversion

Reply:
Sure, I'll change it to default to `None`. As of reason I did that, was mainly because no conversions is pretty useless right now for us, but sure I'll change it.